### PR TITLE
H 1413 custom thanks message new pull request 

### PR DESCRIPTION
**Pull request by horus2021**
I have merge conflicts with old h 1413 pull request. I cannot fix it so I create a new pull request as a quick fix.

### CODE REVIEW

File: src/components/DonateButton.jsx [hunk 1]
```diff
@@ -1082,23 +1082,38 @@ export default class Root extends React.PureComponent {
...
-          title: 'Thank you!',
-          text: 'Would you like to support Giveth as well?',
-          icon: 'success',
-          buttons: ['No Thanks', 'Support Giveth'],
-        }).then(result => {
-          if (result) {
-            this.defaultDacDonateButton.current.openDialog();
-          }
-        });
+        if (typeof customThanksMessage !== 'undefined') {
```
Comment (aminlatifi) [inline src/components/DonateButton.jsx:1093]: `customThanksMessage` is shown only if we have `donateToDefaultDac`

**Review by aminlatifi — CHANGES_REQUESTED** [review]

File: src/components/DonateButton.jsx (hunk 1 above)
Comment (horus2021) [reply src/components/DonateButton.jsx:1093]: donateToDefualtDac means that user has no profile. If he has no profile, we suggest him to fill his profile. For user without profile who donate to custom message campaign we have the following options.
1. show custom message with more info e.g. Common Stack
2. suggest him to fill his profile (as my commit)
3. show more info e.g Common Stack offer and suggest him to fill his profile

It seems to me  (2) is the best option. (3) suggesting two things is complicated UX. (1) is low priority comparing to (2). It is my opinion, what do you think?

Comment (horus2021) [reply src/components/DonateButton.jsx:1093]: - if know user{
-       if custom thanks
-          show custom thanks
-       elsif Donate to Default
-          suggest donate to default 
-        else 
-           show simple thanks
- else //anon user
-     suggest fill profile

Comment (aminlatifi) [reply src/components/DonateButton.jsx:1093]: Yes it's the correct scenario

Comment (horus2021) [reply src/components/DonateButton.jsx]: @aminlatifi  please check out  my new commit.

**Pull request merged**